homeworks/README.md

```diff
@@ -0,0 +1,49 @@
+# Installation instructions
+\
+To complete the assignments, you need a working Python installation with Jupyter and the following libraries: numpy, scipy, matplotlib, 
+scikit-learn, pytorch, torchvision and pillow. We recommend using Anaconda although this is not mandatory.
+
+## Installing Anaconda
+\
+You can find Anaconda installation instructions in the [official Anaconda documentation](https://docs.anaconda.com/anaconda/install/index.html).

```

Comment:
Link to Miniconda instead of Anaconda. Anaconda ships a lot of things which confuses some students.